src/Compatibility/Core/src/iOS/Renderers/ListViewRenderer.cs

```diff
@@ -274,6 +274,9 @@ protected override void OnElementChanged(ElementChangedEventArgs<ListView> e)
 					}
 					_tableViewController = new FormsUITableViewController(e.NewElement, _usingLargeTitles);
 					SetNativeControl(_tableViewController.TableView);
+					
+					if (Forms.IsiOS15OrNewer)
+						_tableViewController.TableView.SectionHeaderTopPadding = new nfloat(0);
```

Comment:
Fix issue with ListView wrong top padding in iOS 15
https://github.com/xamarin/Xamarin.Forms/pull/14671